administration/src/project-configs/bayern/config.ts

```diff
@@ -15,6 +15,7 @@ const config: ProjectConfig = {
   createEmptyCard: createEmptyBavariaCard,
   dataPrivacyHeadline: dataPrivacyBaseHeadline,
   dataPrivacyContent: DataPrivacyBaseText,
+  timezone: 'UTC',
```

Comment:
Why did you choose UTC here? Should be 'Europe/Berlin' I guess?

Reply:
just by mistake, used for testing. thx for the find